common/gpu/intel.nix

```diff
@@ -0,0 +1,16 @@
+{ config, lib, pkgs, ... }:
```

Comment:
I think we include this in all intel laptops at least.

Reply:
Did you mean to comment on `boot.initrd.kernelModules = [ "i915" ];`?

Reply:
I was more thinking about your suggestion here:

> Should all the profiles using common/cpu/intel be modified to also have common/gpu/intel? If the same computer model can sometime have an Nvidia card, it could cause the problem I'm trying to fix.

Since most of devices are laptops, this is a sane assumption. I can only think about some super micro server boards, where we can skip this.